cg/meta/upload/gt.py

```diff
@@ -67,7 +97,7 @@ def analysis_sex(self, qc_metrics_file: Path) -> dict:
     def get_bcf_file(self, hk_version_obj: housekeeper_models.Version) -> housekeeper_models.File:
         """Fetch a bcf file and return the file object"""
 
-        bcf_file = self.hk.files(version=hk_version_obj.id, tags=["snv-gbcf"]).first()
+        bcf_file = self.hk.files(version=hk_version_obj.id, tags=["genotype"]).first()
```

Comment:
get the file not being the index

Reply:
Can you have a look at the new solution for this comment